src/tiles/grid/item-tile.ts

```diff
@@ -32,7 +32,7 @@ export class ItemTile extends LitElement {
           </div>
           <div class="item-creator">
             <div class="truncated">
-              <span><strong>By:&nbsp;</strong>${itemCreator}</span>
+              ${itemCreator ? html`<span>by&nbsp;${itemCreator}</span>` : ''}
```

Comment:
```suggestion
              ${ this.model?.creator ? html`<span>by&nbsp;${itemCreator}</span>` : nothing }
```

nothing is usually preferred.

Reply:
please also add a quick unit test so we can keep coverage up. thank u!

Reply:
Please see: https://lit.dev/docs/api/templates/#nothing

We've been using this as the right "lit.dev" way to have the DOM cleared and empty, as @iisa suggests above.